nautobot/extras/api/views.py

```diff
@@ -441,9 +441,18 @@ def sync(self, request, pk):
             raise CeleryWorkerNotRunningException()
 
         repository = get_object_or_404(GitRepository, id=pk)
-        repository.sync(user=request.user)
-        return Response({"message": f"Repository {repository} sync job added to queue."})
 
+        job_result = repository.sync(user=request.user)
+
+        return Response({
+                "message": f"Repository {repository} sync job added to queue.",
+                "repository_id": repository.id,
+                "repository_name": repository.name,
+                "sync_jobmodel_id": job_result.job_model_id,
+                "sync_jobresult_id": job_result.id,
+                "sync_status": job_result.status
```

Comment:
Since the sync runs asynchronously (😄) the `sync_status` here will almost always just be `pending` at the time we send this response, so I'd suggest omitting it.

Reply:
True! I was aware of the fact it's always `pending` but my thinking was that there may be scenarios, now (not sure?) or future, where it's something other than that.

It's not something I feel strongly about though, so I'm happy to remove it if you don't see merit in that.

Reply:
Dropped it as suggested after more thought.